R/infrastructure.R

```diff
@@ -291,6 +291,27 @@ use_package_doc <- function(pkg = ".") {
   writeLines(out, file.path(pkg$path, path))
 }
 
+# help function for use_package
+# takes the same arguments as use_package
+build_package_txt <- function(package, version = NULL,
+                              compare =  c(">=", ">", "==", "<=", "<")) {
```

Comment:
Not to barrage you with updates, however, I ran a script that checked CRAN's available packages for all dependencies that called on comparators (≤, <, >, or ==). There were [10 packages](https://gist.github.com/stevenpollack/0b97208a3e097dbfc5fc#file-packages_with_odd_comparators-tsv), all of which published after 26.1.2015...

The most surprising fact is that popular use of `==`, with 3 counts. Whereas, `≤` and `<` have a total of 8...